de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/48-de.metas.inoutcandidate/5476620_sys_gh2940_ShipmentDateToday_Parameter_In_GenerateShipments.sql

```diff
@@ -0,0 +1,45 @@
+-- 2017-11-08T14:47:35.219
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,ColumnName,Created,CreatedBy,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,543470,0,'IsShipmentDateToday',TO_TIMESTAMP('2017-11-08 14:47:35','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.inout','Y','IsShipmentDateToday','IsShipmentDateToday',TO_TIMESTAMP('2017-11-08 14:47:35','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
`IsShipmentDateToday` is a process-parameter..it's not used in a table..but i think, from the actual name, it sounds like a flag..imho it should be more "imperative"..like "isShipToday"...

Reply:
Done